packages/reports/src/Constants/data-table-constants.tsx

```diff
@@ -17,7 +17,9 @@ import IndicativeCell from '../Components/indicative-cell';
 import MarketSymbolIconRow from '../Components/market-symbol-icon-row';
 import ProfitLossCell from '../Components/profit_loss_cell';
 import CurrencyWrapper from '../Components/currency-wrapper';
-import { ITransformer } from 'mobx-utils';
+import type { TRootStore } from 'Stores/index';
+
+type TPortfolioStore = TRootStore['portfolio'];
```

Comment:
can we update these to Returntype<typeof useStore>['portfolio'] for consistency?